src/test/java/pages/LoginPage.java

```diff
@@ -1,6 +1,5 @@
 package pages;
 
-import org.openqa.selenium.By;
```

Comment:
Please, create classs BasePage

With a Base Page Object we can abstract all common actions into a central location and leverage them in any Page Object we see fit.

Our BasePage class will have a constructor which takes a WebDriver object to initialize a WebDriverWait object. The constructor will also be responsible to initialize WebElements via PageFactory.

For example **public class LoginPage extends BasePage**